### ISSUE TITLE
Remove HTML 5 functionality helper code

### ISSUE THREAD

**Issue by titusfortner** [SeleniumHQ/selenium#10397]
### Feature and motivation

This is ApplicationCache, WebStorage, & LocationContext

### Usage example

#10344

**Comment by sc-mk** [comment]
Hi Titus,
any suggestions on how to solve the problem with the MoveToElement method, please?

**Comment by titusfortner** [comment]
I think what needs to be decided here is how we are handling all of the HTML5 functionality across all the languages. 

Next step is essentially an inventory of which of the storage/cache/location things are being supported by JS and which have been removed and decide which direction we want for all of the bindings.

**Comment by symonk** [comment]
@titusfortner what do we do in the short term? `driver.application_cache` simply raises and exception right now and will not work in the python side - do we remove it (it's already deprecated) until we come up with the above and possibly implement something else?

**Comment by titusfortner** [comment]
Next step was to figure out which languages marked as deprecated, which removed and which implemented them in JS.

Then we figure out how they should be done in all of them.

**Comment by titusfortner** [comment]
More accurately this should be discussing the HTML functionality that was added back in 2015

ApplicationCache, WebStorage, & LocationContext

Need to inventory which languages have methods pointing to outdated endpoints, which ones have re-implemented with JS and which have deleted them.

**Comment by titusfortner** [comment]
Per [TLC Meeting](https://www.selenium.dev/meetings/2022/tlc-11-10/):

> @diemol will list the changes that need to be done in the bindings and then deprecate/remove them

**Comment by iampopovich** [comment]
Hi 👋
I'm planning to assist with this task. Please let me know if there has been any progress or new discussions about where to start. For now, I intend to familiarize myself with the implemented methods and follow this plan: "Need to inventory which languages have methods pointing to outdated endpoints, which ones have re-implemented with JS and which have deleted them."

**Comment by titusfortner** [comment]
This has been lower priority/mostly forgotten about. Any help would be greatly appreciated!

**Comment by aguspe** [comment]
Hi I would like to help with the ruby bindings for this @titusfortner I will start working on it now and push a PR over the weekend

**Comment by aguspe** [comment]
After the merge of #14276 is there anything we can continue working on regarding this for the ruby bindings? or should we wait until a general decision on all the bindings is done? @titusfortner

**Comment by aguspe** [comment]
Would it be possible to remove the rb label here @p0deje ? 

I'm just looking for something new to pick up and it will help me filter, thank you!

**Comment by juddgaddie** [comment]
While the API in Java has been deprecated what is the new recommended implementation to achieve the same functionality?

**Comment by Delta456** [comment]
I don't see `ApplicationCache`, `WebStorage`, & `LocationContext` in Python anymore so the python tag can be removed.

**Comment by iampopovich** [comment]
The `selenium.html5` and `selenium.remote.html5` packages, as well as HTML5 tests, still exist in the project. Is this task still relevant? 

> While the API in Java has been deprecated what is the new recommended implementation to achieve the same functionality?

Given the lack of response to the [comment](https://github.com/SeleniumHQ/selenium/issues/10397#issuecomment-2366313130), I conclude there is no replacement for the deprecated functionality. Can these packages be deleted, and necessary changes made to dependent classes?

**Comment by diemol** [comment]
@iampopovich, yes, they need to be deleted. You should check if they are already deprecated.

**Comment by iampopovich** [comment]
@Diemol, in the commits above, I removed the leftover commands from the Java and JavaScript bindings. I also removed commands and interfaces for the deleted classes from the Ruby binding. It seems they were forgotten.

These changes are based on the removal of similar commands from the .NET binding.
